custom_components/healthchecksio/binary_sensor.py

```
@@ -40,37 +41,35 @@ async def async_setup_entry(hass, config_entry, async_add_devices):
 
 
 class HealthchecksioBinarySensor(BinarySensorEntity):
```

Pylint (3.3.7)

[refactor] Too many instance attributes (9/7)

(R0902)